sites/aviationpros.com/server/components/flows/content-load-more.marko

```diff
@@ -5,7 +5,6 @@ $ const { aliases } = input;
 $ const nodes = getAsArray(input, "nodes");
 
 <if(nodes.length)>
-  <website-content-card-deck-flow nodes=nodes.slice(0, 10) aliases=aliases native-x=input.nativeX />
   <default-theme-column-list-flow nodes=nodes.slice(10)>
```

Comment:
Shouldn't the slice be reversed now? The list should be `(0, 10)` and the cards `(10)`? Otherwise you'll be showing the content in reverse order...

Reply:
@zarathustra323 - Good catch; changed to slice(0, 4) and slice(4).